deploy-board/deploy_board/templates/clusters/cluster-replacements.tmpl

```diff
@@ -288,13 +408,61 @@
+                            </p>
+                        </div>
+                    </div>
+
+                    <div style="padding-top: 20px;"></div>
+
+                    <div id="minHealthyPercentageBlock">
+                        <label for="minHealthyPercentage" data-toggle="tooltip"
                         title="The percentage of the cluster size that must be in healthy state. E.g. with 90%, each round, 10% of the cluster will be replaced until completion. If minimum healthy percentage results in 0 instance to be replaced, then 1 instance will be replaced at a time. Use 100 if you want to replace 1 instance at time.">
                         <span class="fa fa-check"></span> Minimum healthy percentage:
-                    </label>
-                    <input type="number" id="minHealthyPercentage" name="minHealthyPercentage" min="0" max="100" value="90">
+                        </label>
+                        <input type="number" id="minHealthyPercentage" name="minHealthyPercentage" min="0" max="100" value="90">
+                    </div>
+
+                    <div id="maxHealthyPercentageBlock">
+                        <label for="maxHealthyPercentage" data-toggle="tooltip"
+                        title="The maximum percentage of the group that can be in service and healthy, or pending, to support your workload when replacing instances. The value is expressed as a percentage of the desired capacity of the Auto Scaling group. Value range is 100 to 200. Use 100 to launch/replace 1 instance at time">
+                        <span class="fa fa-check"></span> Maximum healthy percentage:
+                        </label>
+                        <input type="number" id="maxHealthyPercentage" name="maxHealthyPercentage" min="100" max="200">
+                        <p>Example: with 110% max, new hosts equal up to 10% of the cluster will be launched first. After the new hosts are launched and deployed successfully, up to 10% of the original existing hosts will be terminated. For example, the current capacity is 2 hosts, then with 110% max, 1 host will be launched first. The total capacity is now 3.
+                            After the new host is deployed successfully, one of the original 2 hosts will be terminated. This repeats until the original 2 hosts are replaced with 2 new hosts. During the replacement, 100% of the original capacity (2 hosts) are ensured to be available - hence availability is prioritized.
+                            If a new host fails deployment, it will be terminated and will not be put into service. Note that similar to minimum healthy percentage, if it's too small, then 1 host will be launched. Use 100% to ensure that only 1 host will be launched at a time.
+                        </p>
+                    </div>
+                </div>
+                </fieldset>
 
+                <div style="padding-top: 20px;">
```

Comment:
Is this more or less the same as lines 280-330? If so, I think it might be worth consolidating into a separate template to make it easier to update in the future